packages/compass-schema/src/components/coordinates-minichart/coordinates-minichart.jsx

```diff
@@ -278,5 +283,7 @@ class CoordinatesMinichart extends PureComponent {
   }
 }
 
+const CoordinatesMinichart = withDarkMode(UnthemedCoordinatesMinichart);
```

Comment:
When zooming out, the map container shows light color in dark mode.

<img width="300" alt="image" src="https://user-images.githubusercontent.com/1305718/220886579-1b8df365-7083-40ce-80cf-f36e7276a94c.png">